corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -442,6 +442,18 @@ hqDefine('app_manager/js/releases/releases', function () {
         self.revertSavedApp = function (savedApp) {
             $.postGo(self.reverse('revert_to_copy'), {build_id: savedApp.id()});
         };
+        self.handleDeprecatedCaseTypesWarning = function(depCaseTypes) {
+            if (depCaseTypes && depCaseTypes.length) {
+                $('#deprecated-case-types-warning').removeClass('hidden');
```

Comment:
Could use knockout's [visible or hidden binding[s](](https://knockoutjs.com/documentation/visible-binding.html)url) for this usecase